lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -86,9 +90,7 @@ function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL) {
           'crop_rotate' => $values['crop_rotate'],
         ]),
   ];
-
-  $response = drupal_http_request($client['base_url'] . '/reportbacks', $options);
-
+  $response = post($client['base_url'] . '/reportbacks', $options);
```

Comment:
So part of the issue too is that you're not actually instantiating the `RestApiClient` when building the client in https://github.com/DoSomething/phoenix/blob/db40d34f09c1856d3eb5193ff6ed5e8b71eb5e5b/lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module#L38 So you'll pretty much never have access to the `post()` method without it!